frontend/model/contracts/chatroom.js

```diff
@@ -300,10 +258,8 @@ sbp('chelonia/defineContract', {
         Vue.set(state.attributes, 'name', data.name)
 
         const notificationData = createNotificationData(MESSAGE_NOTIFICATIONS.UPDATE_NAME, {})
-        addMessage(state, createMessage({ meta, hash, height, state, data: notificationData, innerSigningContractID }))
-      },
-      sideEffect ({ contractID, hash, meta }) {
-        setReadUntilWhileJoining({ contractID, hash, createdDate: meta.createdDate })
```

Comment:
Why was this deleted?

Reply:
As the name of function shows, it's for setting `readUntil` position of a chatroom.
In the previous way of using Indexed Storage, we set the `readUntil` position to the latest message when the user syncs the chatroom contract for the first time. So, every function that creates a message had this function call in it's `sideEffect` to set the `readUntil` to the message which is created in it's `process`.

But, in the current way of using KV store, `readUntil` position is initialised only in `chatroom/join/sideEffect` function, and can be updated only when the messages are displayed in the screen and user sees them. So, the updating of `readUntil` can't be processed in the background and it's handled in `ChatMain.vue`.